psydac/api/ast/nodes.py

```diff
@@ -775,9 +778,9 @@ def weights( self ):
         return self._weights
 #==============================================================================
 class PlusTensorQuadrature(TensorQuadrature):
+    """This class represents the quadrature point and weight in the plus boundary of an interface.
```

Comment:
Same question here: do you meant "quadrature points and weights" (plural)?

Reply:
here it is singular, because the quadrature point and the weight are fixed

Reply:
See above. I think the class names are confusing

Reply:
done in [20f4c78](https://github.com/pyccel/psydac/pull/160/commits/20f4c788cb999572c4a156bf4eb893e1fc3bdc02)